docs/scala_toolchain.md

```diff
@@ -46,3 +44,54 @@ scala_register_toolchains()
     # WORKSPACE
     register_toolchains("//toolchains:my_scala_toolchain")
     ```
+
+<table class="table table-condensed table-bordered table-params">
```

Comment:
I think you can do a table in github flavored markdown https://help.github.com/en/articles/organizing-information-with-tables

Reply:
I tried converting this to MD, but it's not clear I got any readability wins. In order to insert newlines into cells, I needed to mix MD and HTML. I also cannot indent the HTML properly because a newline starts a new cell.